R/writer.R

```diff
@@ -51,8 +51,8 @@ write_session_info <- function(){
       map_chr(~ ifelse(nchar(.x) > 80 & grepl("\u2500\u2500\u2500\u2500", .x),
                    substring(.x, 1, 80),
                    .x)) %>%
-      # wrap any other elements over 78 characters
-      map_chr(~ stri_wrap(.x, width = 78, exdent = 2, simplify = FALSE,
+      # wrap any other elements over 80 characters
+      map_chr(~ stri_wrap(.x, width = 80, exdent = 2, simplify = FALSE,
```

Comment:
I tried this out and see the line after session ino wraps.  Is this expected?


─ Session info
	  ─────────────────────────────────────────────────────────────────